<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OsagoUsingInfoForm</class>
 <widget class="QDialog" name="OsagoUsingInfoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_9">
   <item>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="1" column="1">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Ранее заключенные договора ОСАГО</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="lastOsagoGridLayout">
          <item row="1" column="1">
           <widget class="QLineEdit" name="lastOsagoSerial"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Страховая
компания</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lastOsagoNum"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Серия</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Номер</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="lastOsagoContractCheckBox">
            <property name="text">
             <string>В отношение данного ТС ранее 
уже заключался договор ОСАГО</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lastOsagoCompany"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_5"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Особые отметки</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Иные сведения</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_4"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>Страховой полис</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_20">
                  <property name="text">
                   <string>Серия</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_6"/>
                </item>
                <item>
                 <widget class="QLabel" name="label_21">
                  <property name="text">
                   <string>Номер</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_7"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>158</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QPushButton" name="prevButton">
           <property name="text">
            <string>Назад</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>428</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="printRequest">
           <property name="text">
            <string>Печать заявления</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="printBlank">
           <property name="text">
            <string>Печать бланка</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Водители, допущенные до управления ТС</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0" colspan="2">
             <widget class="QRadioButton" name="unlimDriversButton">
              <property name="text">
               <string>Любые водители (без ограничений)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="constDriversButton">
              <property name="text">
               <string>Только следующие водители:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="showDriverButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>31</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>31</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="text">
                 <string>+</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="hideDriverButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>31</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>31</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="text">
                 <string>-</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0" colspan="2">
           <layout class="QGridLayout" name="driversGridLayout" rowstretch="0,1,0,0,0">
            <item row="0" column="4">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Стаж</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QComboBox" name="comboBox_4">
              <item>
               <property name="text">
                <string>М</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Ж</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="lineEdit_8">
              <property name="minimumSize">
               <size>
                <width>136</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QLineEdit" name="lineEdit_13"/>
            </item>
            <item row="3" column="1">
             <widget class="QDateEdit" name="dateEdit_6"/>
            </item>
            <item row="2" column="0">
             <widget class="QLineEdit" name="lineEdit_10"/>
            </item>
            <item row="4" column="3">
             <widget class="QLineEdit" name="lineEdit_15"/>
            </item>
            <item row="3" column="4">
             <widget class="QDateEdit" name="dateEdit_7"/>
            </item>
            <item row="4" column="0">
             <widget class="QLineEdit" name="lineEdit_14"/>
            </item>
            <item row="4" column="4">
             <widget class="QDateEdit" name="dateEdit_9"/>
            </item>
            <item row="2" column="2">
             <widget class="QComboBox" name="comboBox_6">
              <item>
               <property name="text">
                <string>М</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Ж</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QComboBox" name="comboBox_7">
              <item>
               <property name="text">
                <string>М</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Ж</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Дата рождения</string>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Класс</string>
              </property>
             </widget>
            </item>
            <item row="4" column="5">
             <widget class="QComboBox" name="comboBox_5"/>
            </item>
            <item row="1" column="1">
             <widget class="QDateEdit" name="dateEdit_2"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>ФИО</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QLineEdit" name="lineEdit_11"/>
            </item>
            <item row="2" column="5">
             <widget class="QComboBox" name="comboBox_2"/>
            </item>
            <item row="2" column="1">
             <widget class="QDateEdit" name="dateEdit_4"/>
            </item>
            <item row="2" column="4">
             <widget class="QDateEdit" name="dateEdit_5"/>
            </item>
            <item row="1" column="5">
             <widget class="QComboBox" name="comboBox"/>
            </item>
            <item row="4" column="2">
             <widget class="QComboBox" name="comboBox_8">
              <item>
               <property name="text">
                <string>М</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Ж</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLineEdit" name="lineEdit_9"/>
            </item>
            <item row="3" column="0">
             <widget class="QLineEdit" name="lineEdit_12"/>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Пол</string>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QDateEdit" name="dateEdit_3"/>
            </item>
            <item row="4" column="1">
             <widget class="QDateEdit" name="dateEdit_8"/>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Водительское удостоверение
 (серия, номер)</string>
              </property>
             </widget>
            </item>
            <item row="3" column="5">
             <widget class="QComboBox" name="comboBox_3"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
       <zorder></zorder>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Период использования ТС</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QGridLayout" name="timeGridLayout">
          <item row="0" column="2" colspan="3">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>158</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QDateEdit" name="dateEdit"/>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QDateEdit" name="dateEdit_12"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>С</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QDateEdit" name="dateEdit_14"/>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>По</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>С</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>По</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>По</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="addTimeButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>31</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>31</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="delTimeButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>31</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>31</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>С</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QDateEdit" name="dateEdit_10"/>
          </item>
          <item row="4" column="1" colspan="2">
           <widget class="QDateEdit" name="dateEdit_16"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>С</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>По</string>
            </property>
           </widget>
          </item>
          <item row="4" column="4">
           <widget class="QDateEdit" name="dateEdit_15"/>
          </item>
          <item row="3" column="4">
           <widget class="QDateEdit" name="dateEdit_13"/>
          </item>
          <item row="2" column="4">
           <widget class="QDateEdit" name="dateEdit_11"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder></zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
